src/tasks/test-deployment.ts

```diff
@@ -58,6 +59,7 @@ const defaultArgs = {
 } as const;
 interface DeployTaskArgs {
   mnemonic: string;
+  claimCsv?: string;
```

Comment:
This could actually be not existing as it comes from Hardhat. Note that it's a different type (`DeployTaskArgs`) to the one with the changes from `?:` to `: string | undefined` (`CleanArgs`).